app/_posts/2024-11-05-kuma-2-9-0.md

```diff
@@ -0,0 +1,107 @@
+- You may want to manage traffic going across zones differently from traffic that is staying inside a zone.
+
+With `MeshService` we now have a dedicated service object which selects a set of data-planes to send traffic to.
+This `MeshService` is defined in both policies and envoy configuration, making it very easy to understand and observe.
+The resources can be labeled which enables policies that applies to whole subset of your services.
+
+[`MeshService`](/docs/2.9.x/networking/meshservice/) are cross-zone by default but not multi-zone.
+This means that each `MeshService` can only be present in a single zone at time.
+However, it can be accessed by other zones thanks to our cross-zone communication setup.
+If you want a service to be multi-zone you need to create a [`MeshMultizoneService`](/docs/2.9.x/networking/meshmultizoneservice/) which will group multiple services inside the same "pseudo-service".
+
+Along with `MeshService` we're rolling new ways to address services, this means that we're parting with the `.mesh` addresses that were used in the past we will now
+use addresses that by default are zone and service type specific. This is mostly to make the getting started experience easier and this is configurable for users.
+This is configurable with [`HostnameGenerator`](/docs/2.9.x/networking/hostnamegenerator/)
+
+`MeshService` is currently disabled by default and there's a migration procedure if you want to use it.
+Because the use cases and usability of Kuma is greatly improved we've updated our quickstart to use `MeshService`.
+If you want to look at migration check out our  [migrating to mesh service doc](/docs/2.9.x/networking/meshservice/#migration). 
+
+We're also working on a comprehensive guide to help get a handle on `MeshService` and `MeshMultizoneService`.
+
+## Namespaced policies and policy attachment rework for outbound policies
+
+One of the thing that Kuma excels out is its policy system.
+Its goal is to be as powerful and expressive as possible and to respond to both of our user's personas: The mesh operator and the service owner.
+
+One part of our vision for policies is to "make policies as easy as Horizontal Pod autoscaler" meaning by this each service owner should be
+able to just add their policies in their helm chart.
+Up until now this was problematic because policies needed to be applied to the mesh system namespace.
+This is no longer the case with namespaced policies where policies can be created in any namespace on a Kubernetes cluster.
+
+This also opened up 2 types of policies:
+
+- Policies that impacts services that I do control, in this case I should be able to add to these policies configuration that impacts the clients. Because, as the service owner I know best what to do.
```

Comment:
can we simplify this point somehow? it is hard to understand what this policy should do